dbcon/execplan/mcsanalyzetableexecutionplan.h

```diff
@@ -0,0 +1,170 @@
+/* Copyright (C) 2021 MariaDB Corporation
+
+   This program is free software; you can redistribute it and/or
+   modify it under the terms of the GNU General Public License
+   as published by the Free Software Foundation; version 2 of
+   the License.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+   MA 02110-1301, USA. */
+
+#ifndef MCSANALYZETABLEXEXCUTIONPLAN_H
+#define MCSANALYZETABLEXEXCUTIONPLAN_H
+#include <vector>
+#include <map>
+#include <iosfwd>
+
+#include <boost/uuid/uuid.hpp>
+
+#include "exp_templates.h"
+#include "calpontexecutionplan.h"
+#include "calpontselectexecutionplan.h"
+#include "returnedcolumn.h"
+#include "filter.h"
+#include "expressionparser.h"
+#include "calpontsystemcatalog.h"
+#include "brmtypes.h"
+#include "objectreader.h"
+#include <boost/algorithm/string/case_conv.hpp>
+
+/**
+ * Namespace
+ */
+namespace execplan
+{
+
+class MCSAnalyzeTableExecutionPlan : public CalpontExecutionPlan
+{
+  public:
+    typedef std::vector<SRCP> ReturnedColumnList;
```

Comment:
Could you make these members private.

Reply:
Sure.

Reply:
Oops, probably no, because I use them to define a type in plugin code, before creating exec plan, because exec plan is initialized by `ReturnedColumnList` and `ColumnMap`.